Makefile.inc1

```diff
@@ -895,17 +895,14 @@ XCFLAGS+=	${BFLAGS}
 
 .if ${MK_COMPAT_CHERIABI} == "yes"
 _LIBCOMPAT=CHERI
-.include "Makefile.libcompat"
 .elif ${MK_LIB32} == "yes"
 _LIBCOMPAT= 32
-.include "Makefile.libcompat"
 .elif ${MK_LIB64} == "yes"
 _LIBCOMPAT=64
-.include "Makefile.libcompat"
 .elif ${MK_LIBSOFT} == "yes"
 _LIBCOMPAT= SOFT
-.include "Makefile.libcompat"
 .endif
+.include "Makefile.libcompat"
```

Comment:
I wonder if this want's to be:
```suggestion
.if defined(_LIBCOMPAT)
.include "Makefile.libcompat"
.endif
```

Reply:
No, because the recursive make invocation I've added won't have _LIBCOMPAT defined, instead all the variables will be as if it's a "normal" build for libcheri/lib32/lib64 (as is done for the recursive call for `libraries`). I don't need most of `Makefile.libcompat`, but I do need the bit that I put outside `.if defined(_LIBCOMPAT)`. That could live outside of `Makefile.libcompat` itself, but I felt it was nicer to keep it contained within the file.

Reply:
Makes sense.